kernel_gateway/tests/test_gatewayapp.py

```diff
@@ -92,7 +92,7 @@ def test_ssl_options(self):
         self.assertEqual(ssl_options['ssl_version'], 5)
 
     def test_load_notebook_local(self):
-        nb_path = os.path.join(RESOURCES, 'weirdly?named#notebook.ipynb')
```

Comment:
This file name causes an error while cloning the repo on a Windows machine - NTFS does not allow to use `?` in file names.
So I've renamed it.